extensions/deploy_replace_undeploy/standard/requirements/deploy-replace-undeploy/package/REQ_get-op.adoc

```diff
@@ -0,0 +1,9 @@
+[[req_deploy-replace-undeploy_package_get-op]]
+[requirement]
+====
+[%metadata]
+label:: /req/deploy-replace-undeploy/package/get-op
+part:: For every dynamically added process (path: /processes/{processId}), the server SHALL support the HTTP GET operation at the path `/processes/{processId}/package`.
```

Comment:
Include `POST` along with `path: /processes/{processId}` to avoid ambiguity with `GET` from core.